python/snewpy/snowglobes.py

```diff
@@ -343,6 +343,7 @@ def simulate(SNOwGLoBESdir, tarball_path, detector_input="all", verbose=False):
     sng = SNOwGLoBES(SNOwGLoBESdir)
     if detector_input == 'all':
         detector_input = list(sng.detectors)
+        detector_input.remove('d2O')
```

Comment:
Is there any context in which we would want `d2O` included in `sng.detectors`? If not, it might make more sense to remove it in `_load_detectors()` (in `snowglobes_interface.py`), so we don’t have to do this everytime we iterate over other detectors?
(E.g. I note that there’s similar code in `collate()`; though I’ll have a separate PR on that in a moment …)

Reply:
We might. In case someone adds the missing cross-sections to SNOwGLoBES, the interface will be working.
Probably the proper way to solve this would make a check in `_load_detectors` an drop the detector if the cross-sections are missing.